tgui/packages/tgui/interfaces/BankingComputer/type.ts

```diff
@@ -0,0 +1,73 @@
+/**
+* @file
+* @copyright 2024
+* @author mloccy (https://github.com/mloccy)
+* @license MIT
+*/
+
+import { SortDirection } from "../PlayerPanel/constant";
+export const CREDIT_SIGN = "⪽";
```

Comment:
Can we get this stuck into `.../interfaces/common/strings.ts` or something? Follow-up work from that (for a different PR): change usages to have them all point there.